services/workflows-service/src/events/workflow-completed-webhook-caller.ts

```diff
@@ -120,4 +126,36 @@ export class WorkflowCompletedWebhookCaller {
       },
     });
   }
+
+  private async updateSalesforceRecord(workflowRuntimeData: WorkflowRuntimeData) {
+    const { salesforceRecordId, salesforceObjectName } = workflowRuntimeData;
+
+    if (!salesforceRecordId || !salesforceObjectName) {
+      return;
+    }
+
+    const statusMap = {
```

Comment:
Thoughts about using an array here instead? Could use `some` or `includes` and such. Most of our enums also export a type and an array.